sdk/storage/src/blob/blob/responses/delete_blob_response.rs

```diff
@@ -11,7 +11,7 @@ response_from_headers!(DeleteBlobResponse ,
 
 #[cfg(feature = "azurite_workaround")]
 response_from_headers!(DeleteBlobResponse ,
-               delete_type_permanent_from_headers => delete_type_permanent: Option<bool>,
+               delete_type_permanent_from_headers => delete_type_permanent: bool,
```

Comment:
Can you explain why you made this change? I don't believe this has anything to do with moving table service to the pipeline architecture.

Reply:
I forgot to undo the change. without removing the option I could not able to build the project.
I don't know what is the reason, but after I changed it back its seems to work properly.